testng-core/src/main/java/org/testng/internal/invokers/TestMethodWorker.java

```diff
@@ -109,32 +109,32 @@ public String toString() {
   public void run() {
     this.currentThreadId = Thread.currentThread().getId();
     if (RuntimeBehavior.enforceThreadAffinity()
-        && doesTaskHavePreRequistes()
+        && doesTaskHavePreRequisites()
         && currentThreadId != threadIdToRunOn) {
       completed = false;
       return;
     }
 
-    for (IMethodInstance testMthdInst : m_methodInstances) {
-      ITestNGMethod testMethod = testMthdInst.getMethod();
+    for (IMethodInstance testMethdInstance : m_methodInstances) {
```

Comment:
I think you can add the last missing 'o' 😉

Reply:
Oops :) 
Fixed.